cmd/minishift/cmd/delete.go

```diff
@@ -50,11 +48,6 @@ func runDelete(cmd *cobra.Command, args []string) {
 }
 
 func init() {
-	deleteCmd.Flags().String(username, "", "Username for the virtual machine unregistration.")
-	deleteCmd.Flags().String(password, "", "Password for the virtual machine unregistration.")
-
-	deleteCmd.Flags().AddFlagSet(subscriptionManagerFlagSet)
-
```

Comment:
Removing this because viper need different flagset and variable for different command.